### PR TITLE
feat[WEB-4]: add footer components with styling

### PR DESCRIPTION
**Pull request by sofiiavoitkova**

### CODE REVIEW

File: src/components/Footer/Components/Logo/logo.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styles from './logo.module.scss'
+import logoTitle from "../../../../assets/logoTitle.svg"
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Logo/logo.jsx]: use aliases

File: src/components/Footer/Components/Navigation/navigation.module.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.footerNavigations {
+    font-size: 1rem;
+    line-height: 1.6;
+    letter-spacing: 0.02em;
+    font-family: var(--common-font-en), sans-serif;
+    width: 50%;
+}
+
+.footerNavigations ul {
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Navigation/navigation.module.scss]: better to use classes, don't use tag names

File: src/components/Footer/Components/Socials/socials.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import styles from './socials.module.scss'
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Socials/socials.jsx]: the same, place styles in the end

File: src/components/Footer/footer.module.scss [hunk 4]
```diff
@@ -1,8 +1,24 @@
...
+  position: relative;
   bottom: 0;
-  left: 0;
   width: 100%;
-  height: 100px;
-  background: yellow;
+  padding: 110px 0px 0px;
+  text-align: center;
+  color: var(--common-color-text-inverse-primary);
+  background: linear-gradient(rgb(22, 33, 39) 0%, rgb(0, 0, 0) 100%);
```
Comment (AndrZakhara) [inline src/components/Footer/footer.module.scss]: put gradient to the scss variables

File: src/components/Footer/index.jsx [hunk 5]
```diff
@@ -1,10 +1,24 @@
 import styles from './footer.module.scss'
+import Logo from '../../components/Footer/Components/Logo/logo'
```
Comment (AndrZakhara) [inline src/components/Footer/index.jsx]: use aliases

File: src/components/Footer/index.jsx [hunk 6]
```diff
@@ -1,10 +1,24 @@
...
+import Navigation from '../../components/Footer/Components/Navigation/navigation'
+import Copyright from './Components/Copyright/copyright'
 
-export default function Footer() {
+function Footer() {
   return (
-    <footer className={styles.container}>
-      <div>Title</div>
-      <div>Content</div>
+    <footer className={styles['footer']}>
```
Comment (AndrZakhara) [inline src/components/Footer/index.jsx]: use class names in this way: className={styles.footer}, square brackets isn't needed

File: src/components/Footer/index.jsx [hunk 7]
```diff
@@ -1,10 +1,24 @@
...
+      <div className={styles['footerContainer']}>
```
Comment (AndrZakhara) [inline src/components/Footer/index.jsx]: u can use class 'container' and 'content'

File: src/components/Footer/Components/Navigation/navigation.jsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
+import styles from './navigation.module.scss'
+
+function Navigation() {
+  return (
+    <div className={styles['footerNavigations']}>
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Navigation/navigation.jsx]: square brackets are not needed

File: src/components/Footer/index.jsx [hunk 9]
```diff
@@ -1,10 +1,24 @@
 import styles from './footer.module.scss'
+import Logo from '../../components/Footer/Components/Logo/logo'
+import Socials from '../../components/Footer/Components/Socials/socials'
```
Comment (AndrZakhara) [inline src/components/Footer/index.jsx]: let's use index.jsx as root file in the components

**Review by MyHKo — APPROVED** [review]

File: src/components/Footer/Components/Logo/index.jsx [hunk 10]
```diff
@@ -0,0 +1,14 @@
+import logoTitle from "../../../../assets/logoTitle.svg"
```
Comment (AndrZakhara) [inline src/components/Footer/Components/Logo/index.jsx]: use aliases

**Review by AndrZakhara — APPROVED** [review]

**Pull request merged**